exercises/Laisha-Calalpa/answers.js

```diff
@@ -47,25 +43,13 @@ function shortLongShort(first, second) {
 // Question 8
 function describeAge(age){
   let message = "";
-  message = (age <= 12) ? "You're a kid": (age >= 13 && age <=17) ? "You're a teenager" :  (age >= 18 && age <= 64) ? "You're an adult":  (age >= 65) ? "You're an elderly": false
-  return(message);
+  return age <= 12 ? "You're a kid": age <=17 ? "You're a teenager" :  age <= 64 ? "You're an adult":  age >= 65 ? "You're an elderly": false
```

Comment:
This function should never return `false`. Small edit below:
```javascript
return age <= 12 ? "You're a kid": age <=17 ? "You're a teenager" :  age <= 64 ? "You're an adult":  "You're an elderly"
 ```
Moral of the story, you don't need to check if `age >= 65` because you already checked if `age <= 64`. If you're not 64 or younger, then you're 65 or older, right?